x-pack/plugins/index_management/public/hooks/use_details_page_mappings_model_management.ts

```diff
@@ -34,9 +34,9 @@ const getCustomInferenceIdMap = (
       ? {
           trainedModelId: model.service_settings.model_id,
           isDeployable: model.service === Service.elser || model.service === Service.elasticsearch,
-          isDeployed: modelStatsById[model.inference_id]?.state === 'started',
+          isDeployed: modelStatsById[model.service_settings.model_id]?.state === 'started',
```

Comment:
I don't think this is correct. We should be looking at the inference ID, because we don't care about the model in general, we care about the deployment of the model that is specific to the inference ID.